### DISCUSSION TITLE
Docker-compose

### DISCUSSION THREAD

**Discussion by ippocratis** [rpdrewes/whisper-websocket-server#3] [Ideas]
I like self hosting my services
Also like the docker philosophy
it would be great if I saw this dockerised
Thanks

**Comment by rpdrewes** [comment]
Yes, I've thought about doing this. It would not be too hard, I think. Have you tried the system it in its current form and do you like it? If there was clear demand I would work up a dockerized system.

**Comment by rpdrewes** [comment]
I have added instructions for creating a docker image.
